Source/Manager/WebSessionRoutes.swift

```diff
@@ -63,6 +63,7 @@ public class WebSessionRoutes {
      - parameter redirectQueryItems: any query items you want the redirect url to contain
      */
     public func forgotPasswordURL(withRedirectPath path: String? = nil, redirectQueryItems: [URLQueryItem]? = nil) -> URL {
+        Settings.setValue(path, forKey: ClientConfiguration.RedirectInfo.ForgotPassword.settingsKey)
```

Comment:
Ah, this was just put back because a previous commit I removed it (if you click the "files changed" tab on this PR this won't be show as added). This is used by AppLaunchData to parse the deep links. But it's a pretty ugly approach :/

Reply:
Fair enough 🙂